.github/workflows/graphql.yml

```
@@ -0,0 +1,26 @@
+name: GraphQL API Documentation
+on: [push, pull_request]

```

```suggestion
on:
  push:
    branches:
      - 'master'
    paths:
      - 'internal/graphql/schema/schema.graphql'   
```
@amino19 Let's modify this a bit so that it triggers only when the graphql schema is changed.
Something similar on pull_request events as well.

---

Some great feedback happening in this review.